ambry-store/src/main/java/com.github.ambry.store/IndexSegment.java

```diff
@@ -780,8 +786,7 @@ private void readFromFile(File fileToRead, Journal journal) throws StoreExceptio
     logger.info("IndexSegment : {} reading index from file", indexFile.getAbsolutePath());
     index.clear();
     CrcInputStream crcStream = new CrcInputStream(new FileInputStream(fileToRead));
```

Comment:
Use 'BufferedInputStream' to avoid small IO?

Reply:
I think in this particular case its not required because this happens only on startup